code/modules/cargo/exporters.dm

```diff
@@ -0,0 +1,36 @@
+/obj/item/weapon/cargo_exporter
+	name = "shipping exporter remote"
+	desc = "Hit objects with this to export them."
+	icon = 'goon/icons/obj/cargo_pad.dmi'
+	icon_state = "exporter"
+	var/works_from_distance = 0
+
+/obj/item/weapon/cargo_exporter/afterattack(atom/movable/T, mob/living/carbon/human/user, flag, params)
+	if(flag)
+		export(T, user)
+		return
+	else
+		if(works_from_distance)
+			if(istype(T) && get_dist(T,src) <= 7)
```

Comment:
Rather than a magic number; make works_from_distance also operate as the range variable of the exporter